game.rb

```diff
@@ -0,0 +1,17 @@
+require './item'
+
+# create game class
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+    super
+  end
+
```

Comment:
**[OPTIONAL]** If you take a careful look at the [UML class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png), The `#can_be_archived?` method is signified with a minus sign meaning it should be a private method.

You could consider adding the `private` in line 12 to apply this.